Source/WebCore/css/SelectorCheckerTestFunctions.h

```diff
@@ -590,27 +590,4 @@ ALWAYS_INLINE bool matchesActiveViewTransitionPseudoClass(const Element& element
     return !!element.document().activeViewTransition();
 }
 
-ALWAYS_INLINE bool matchesActiveViewTransitionTypePseudoClass(const Element& element, const FixedVector<AtomString>& types)
-{
```

Comment:
I don't think we should move this method. We'll want to re-use this for CSSJIT support at some point, and it's generally inconsistent with other methods.

Reply:
I concur (well, I have a CSSJIT patch for `view-transition-type`, but it keeps crashing and I haven't figured it out yet...). If this doesn't affect build speed, I'd like to keep `matchesActiveViewTransitionTypePseudoClass` in `SelectorCheckerTestFunctions.h`